uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/WorkDayTest.php

```diff
@@ -168,10 +168,22 @@ public function testNumberOfHoursShouldBeCalculatedFromTimesAndBreak($timeFrom,
     $this->assertEquals($expectedNumberOfHours, $entity->number_of_hours);
   }
 
+  public function testGetNonWorkingDayTypeValueReturnsTheOptionValueValueAsString() {
+    $this->assertSame('1', WorkDay::getNonWorkingDayTypeValue());
+  }
```

Comment:
Rather than assert against a value can we do something like
`$this->assertSame($this->WorkDayTypes['Weekend']['id'], WorkDay::getNonWorkingDayTypeValue());` ?